node_modules/gh-events/lib/restmodel.js

```diff
@@ -56,21 +56,6 @@
 /**
  * @RESTModel EventOrganiserUpdate
  *
- * @Required []
- * @Property  {EventOrganiserUpdateOther}   organiserOther      Used when the organiser is not a recognised user
- * @Property  {EventOrganiserUpdateUser}    organiserUsers      Used when the organiser(s) is a recognised user
- */
-
-/**
- * @RESTModel EventOrganiserUpdateOther
- *
- * @Required  []
- * @Property  {boolean}                     {organiserUser}     Whether the named user should be an organiser
- */
-
-/**
- * @RESTModel EventOrganiserUpdateUser
- *
  * @Required  []
- * @Property  {boolean}                     {organiserOther}    Whether the named user should be an organiser
+ * @Property  {boolean}                     {organiserUser}     Whether the user should be an organiser
```

Comment:
Is there any reason why we're moving away from having 2 separate parameters for normal and other organisers? That feels like a cleaner REST API to me.